config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml

```diff
@@ -0,0 +1,25 @@
+presubmits:
+  kubernetes-sigs/kubebuilder:
+  - name: pull-kubebuilder-test
+    decorate: true
+    always_run: true
+    optional: false
```

Comment:
I'd suggest to set this to `true` until it was verified with https://github.com/kubernetes-sigs/kubebuilder/pull/689 and the latter was merged

Reply:
fixed